assert_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'rspec'
+require 'selenium-webdriver'
+require "watir"
+
+browser = Watir::Browser.new :chrome 
+
+RSpec.configure do |config|
+  config.color = true
+  config.formatter = :documentation
+  config.before(:each) { @browser = browser }
+  config.before(:all) {browser.window.maximize}
+  config.after(:suite) { browser.close unless browser.nil? }
+end
+
+
+
+describe 'Navigator test' do
+	context "When searching for a place" do
+
+
+it 'should open the main page' do
+ @browser.goto('http://www.navigator.ba/#/categories')

```

Comment:
This step should be divided into two steps. In first one, script will open 'navigator.ba' and second one will be assertion checking if page is correctly redirected to 'http://www.navigator.ba/#/categories'

Reply:
Also done